src/app/_components/PageWrapper.tsx

```diff
@@ -88,7 +88,7 @@ export function PageWrapper({
           minHeight={'100vh'}
         >
           <NavBar tokenPrice={tokenPrice} />
-          <Flex direction={'column'} mt={10} mb={8} gap={7}>
+          <Flex direction={'column'} mt={30} mb={8} gap={7}>
```

Comment:
@andresgalante I removed the mt from PageTitle and added it here as I needed the PageTitle to not have any margin as it was making adding additional UI with the PageTitle difficult